cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -208,7 +220,8 @@ class HydratorPlusPlusTopPanelCtrl{
       startStages: [],
       endStages: [],
       // useSinks: [], // we are not using sinks for now
-      numOfRecords: 25
+      numOfRecords: 25,
```

Comment:
I think we do not need this for preview right now. The numOfRecords is actually not part of preview config, it is part of ETLconfig in the backend